syndeo-template/src/test/java/com/google/cloud/syndeo/perf/SyndeoLoadLT.java

```diff
@@ -0,0 +1,227 @@
+                        SchemaRegistry.createDefault()
+                            .getToRowFunction(
+                                PubsubLiteWriteSchemaTransformProvider
+                                    .PubsubLiteWriteSchemaTransformConfiguration.class)
+                            .apply(
+                                PubsubLiteWriteSchemaTransformProvider
+                                    .PubsubLiteWriteSchemaTransformConfiguration.builder()
+                                    .setFormat("AVRO")
+                                    .setTopicName(topicName)
+                                    .setLocation(LOCATION)
+                                    .setProject(PROJECT)
+                                    .build())))
+                .buildTransform());
+    PipelineResult generatorResult =
+        dataGenerator.runWithAdditionalOptionArgs(
+            List.of("--runner=" + testConfig.getRunner(), "--experiments=enable_streaming_engine"));
+
+    // Build JSON configuration for the template:
+    String jsonPayload =
+        new ObjectMapper()
+            .writeValueAsString(
+                Map.of(
+                    "source",
+                    Map.of(
+                        "urn",
+                        "beam:schematransform:org.apache.beam:pubsublite_read:v1",
+                        "configurationParameters",
+                        Map.of(
+                            "subscriptionName",
+                            subscriptionName,
+                            "location",
+                            LOCATION,
+                            "project",
+                            PROJECT,
+                            "dataFormat",
+                            "AVRO",
+                            "schema",
+                            AvroUtils.toAvroSchema(SyndeoLoadTestUtils.SIMPLE_TABLE_SCHEMA)
+                                .toString())),
+                    "sink",
+                    Map.of(
+                        "urn",
+                        "syndeo:schematransform:com.google.cloud:bigtable_write:v1",
+                        "configurationParameters",
+                        Map.of(
+                            "projectId",
+                            PROJECT,
+                            "tableId",
+                            bigTableName,
+                            "instanceId",
+                            bigtable.getInstanceId(),
+                            "keyColumns",
+                            List.of("sha1")))));
+
+    SyndeoTemplate.buildPipeline(syndeoPipeline, SyndeoTemplate.buildFromJsonPayload(jsonPayload));
+    PipelineResult syndeoResult =
+        syndeoPipeline.runWithAdditionalOptionArgs(
+            List.of("--runner=" + testConfig.getRunner(), "--experiments=enable_streaming_engine"));
+    generatorResult.waitUntilFinish();
+
+    // TODO(pabloem): Handle cancelling the Syndeo Pipeline based on throughput/processing metrics.
```

Comment:
do we also need to check for some results/metrics to determine if the test ran successfully as expected?

Reply:
I've added this check. Thanks.

And yes, this has run on dataflow